### PR TITLE
Empty string check when setting a property.

### PR DESCRIPTION
**Pull request by jose-gomez-evinex**
Update BaseType.php
Added empty string check when setting a property.
Fixes issue #151

### CODE REVIEW

File: src/BaseType.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ public function getType(): string
 
     public function setProperty(string $property, $value)
     {
-        if ($value !== null) {
+        if ($value !== null && $value !== ''){
```
Comment (Gummibeer) [inline src/BaseType.php:29]:
```suggestion
        if ($value !== null && $value !== '') {
```

**Review by Gummibeer — CHANGES_REQUESTED** [review]
Thanks for your PR!
The changes should be done in the generator template as they will otherwise be reverted the next time we generate the source code.

**Comment by jose-gomez-evinex** [comment]
Thanks and sorry about that. I should have be more familiar with the project before submiting any request. Also, I'm still learning how Github works 😅.

**Review by Gummibeer — APPROVED** [review]

**Comment by Gummibeer** [comment]
Everything fine! 😉 That's why we review PRs before merging. 😊

Thanks for the quick fix. I'm on the road right now - but seems good.
Will check in detail, merge and release later. 🎉

**Review by Gummibeer — APPROVED** [review]

**Comment by Gummibeer** [comment]
https://github.com/spatie/schema-org/releases/tag/3.1.1

**Pull request merged**